src/styles/ftva/_block-card-with-image.scss

```diff
@@ -51,6 +51,39 @@
         }
     }
 
+    :deep(.rich-text) {
+        @include ftva-body-2;
+        line-height: 1.5;
+        max-width: 100%;
+        position: relative;
+
+        h3 {
+            @include ftva-h3;
+            color: $body-grey;
+            font-size: 30px;
+        }
+    }
+
+    .custom-title {
+        :deep(.rich-text) {
+            max-width: none;
+            margin: -20px 0px;
```

Comment:
@jendiamond Rather than use negative values on RichText's margins, remove/adjust the margins on RichText's inner elements.